pkg/controllers/user/logging/utils/embedded.go

```diff
@@ -201,6 +210,120 @@ func RemoveEmbeddedTarget(dep rv1beta2.DeploymentInterface, sa rv1.ServiceAccoun
 	return nil
 }
 
+func UpdateEmbeddedEndpoint(podLister rv1.PodLister, serviceLister rv1.ServiceLister, clusterLoggings v3.ClusterLoggingInterface) {
+	timeout := time.After(2 * time.Minute)
```

Comment:
@aiwantaozi can you check if  https://github.com/rancher/rancher/blob/master/pkg/ticker/ticker.go can be used?